pkg/trafficpolicy/types.go

```diff
@@ -154,7 +154,12 @@ type MeshClusterConfig struct {
 	EnableEnvoyActiveHealthChecks bool
 
 	// UpstreamTrafficSetting is the traffic setting for the upstream cluster
+	// +optional
 	UpstreamTrafficSetting *policyv1alpha1.UpstreamTrafficSetting
+
+	// Protocol to use for the cluster
```

Comment:
can you add the set of possible options to the comment?

Reply:
Done.